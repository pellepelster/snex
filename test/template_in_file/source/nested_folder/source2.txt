source 2.1
<!-- snippet:snippet1 -->
old content
<!-- /snippet:snippet1 -->
source 2.2
<!-- snippet:snippet2 -->
<!-- /snippet:snippet2 -->
source 2.3
