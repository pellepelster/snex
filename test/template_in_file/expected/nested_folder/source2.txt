source 2.1
<!-- snippet:snippet1 -->

snippet1.txt#L1-L3"

snippet 1 content

<!-- /snippet:snippet1 -->
source 2.2
<!-- snippet:snippet2 -->

nested_folder/snippet2.txt#L5-L6"

snippet 2 content

<!-- /snippet:snippet2 -->
source 2.3
